src/v2/cookbook/dockerize-vuejs-app.md

```diff
@@ -66,7 +66,7 @@ Let's refactor our `Dockerfile` to use NGINX:
 FROM node:lts-alpine as build-stage
 WORKDIR /app
 COPY package*.json ./
-RUN npm install
+RUN npm install --production
 COPY . .
 RUN npm run build
```

Comment:
How does `npm run build` work if only the production dependencies have been installed?

Reply:
As of my own experience, a normal `npm run build` fails, if only the production dependencies are installed. The build command depends on the dev dependencies and therefore they have to be installed. I think it would be more clear to remove the `--production` option. Inexperienced users may be confused about the build errors.

Reply:
Yeah, I agree.  I messed up.  I can create the pull request to get it back.

Reply:
https://github.com/vuejs/vuejs.org/pull/2392